ProcessMaker/Models/CallActivity.php

```diff
@@ -227,7 +227,7 @@ public function addToken(ExecutionInstanceInterface $instance, TokenInterface $t
      */
     private function syncronizeInstances(ExecutionInstanceInterface $instance, ExecutionInstanceInterface $currentInstance)
     {
-        if ($instance->process->id !== $currentInstance->process->id) {
+        if ($instance->getProcess()->getOwnerDocument()->getModel()->id !== $currentInstance->getProcess()->getOwnerDocument()->getModel()->id) {
```

Comment:
This is a long chain. Is it possible any of these could fail and cause a `Call to a member function on null` or `Trying to get a property of non object` error?

Reply:
@ryancooley thanks for the comment,
`getProcess()` could be null if $instance is created without a process or if the bpmn process ID is invalid.
if getProcess() is valid: `getProcess()->getOwnerDocument()->getModel()` will always hava a valid reference.
I will add a validation